R/g_events_term_id.R

```diff
@@ -121,7 +122,7 @@ g_events_term_id <- function(term,
                              diff_ci_method =
                                c("wald", "waldcc", "ac", "score", "scorecc", "mn", "mee", "blj", "ha", "beal"),
                              axis_side = c("left", "right"),
-                             color = c("blue", "red"),
+                             color = NULL,
```

Comment:
this could be shorter than the condition below
```suggestion
                             color = c(getOption("ggplot2.discrete.colour"), "blue", "red")[1:2],
```